.ruby-version

```diff
@@ -1 +1 @@
-2.2.3
+2.2.0-preview1
```

Comment:
D'oh!  Why did this get kicked down to an older release of Ruby?  I don't believe we want that.